Server/mods/deathmatch/logic/packets/CPedSyncPacket.cpp

```diff
@@ -12,14 +12,10 @@
 #include "StdInc.h"
 #include "CPedSyncPacket.h"
 
-CPedSyncPacket::~CPedSyncPacket()
```

Comment:
Just wondering why the destructor was removed?

Reply:
Shouldn't have been, looks like it went wrong when resolving the merge conflicts

I'll go through it

Reply:
All is good, it's actually not necessary anymore.